resources/keywords/import_provider_account.resource

```diff
@@ -322,8 +334,10 @@ Import Provider Account Failure For DBaaS Policy
     Wait Until Page Contains Element    ${errCreateDBaaSPolicy}     timeout=20s
 
 User Navigates To Import ${databaseProvider} Provider Account On Developer View
-    User Navigates To Add Topology Screen From Developer View      ${databaseProvider}      ${DBaaSTestNS}
-    User Navigates To Import Provider Account Screen From Developer View
+    Wait Until Page Contains Element    ${txtNoDBInstance}    timeout=15s
```

Comment:
@ikanias The existing keywords should not be removed for the success of other test cases. I hope this section has been included to mitigate No database instance error message, which can be handled with conditional block.